recipes/_configure_server.rb

```diff
@@ -105,7 +105,7 @@
 
 license_group =
   case node['splunk']['node_type']
-  when :license_server, :cluster_master, :cluster_slave
+  when :cluster_master, :cluster_slave, :heavy_forwarder, :license_server
```

Comment:
Had misunderstood a wiki page. Just going with the Forwarder license group limits us to not use authentication or indexing on the heavy forwarder which I think is fine for now and could be updated later if we want to enable those things.